tools/apidoc/gen_toc.py

```diff
@@ -245,10 +245,10 @@ def choose_category(fn):
 
 def xml_for(command):
     name = command['name']
-    async = command['async'] and ' (A)' or ''
+    asyncmethod = command['async'] and ' (A)' or ''
```

Comment:
@DaanHoogland shouldn't the header of this python file be changed from `#!/cygdrive/c/Python27` to `#!/cygdrive/c/Python3`?

Reply:
Sorry if this comment is redundant, I see that @nvazquez mentioned about changing other pieces as well.

Reply:
I think the cygwin specific python is not a good idea in any case `#! /bin/env python` would be better.
I think this is being taken into account in ther PRs so maybe i should close, @nvazquez @GabrielBrascher ?